etrago/tools/plot.py

```diff
@@ -1565,8 +1565,9 @@ def plot_grid(self,
         link_colors = pd.Series(data=0, index=network.links.index)
     else:
         logger.warning("line_color {} undefined".format(line_colors))
-
+       
```

Comment:
There are some line with white space which are not needed. They have no effect, but we try to follow some code style rules which don't like these white spaces. 
You could use a tool called `black` to automatically delete them. Did you already used that in eGon-data? If not, I can show you how that works.

Reply:
I just pushed the changes